modules/functions.php

```diff
@@ -450,6 +457,9 @@ function currencySymbol($currency)
     case 'banano':
       return "BANANO";
     
+    case 'nano-beta':
+      return "\u{3B2}NANO";
```

Comment:
Not sure about support of this feature (unicode-symbol injecting) across PHP versions. Update if needed.

Reply:
Should be OK with PHP 7 and above ... I guess we will just keep that as is :)